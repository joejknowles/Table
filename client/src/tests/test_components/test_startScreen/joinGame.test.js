import Component from '../../../components/startScreen/joinGame';

import React from 'react';

import ReactTestUtils from 'react-addons-test-utils';
const shallowRenderer = ReactTestUtils.createRenderer();

it('component renders as before', () => {
  const component = shallowRenderer.render(<Component />);
  expect(component).toMatchSnapshot();
});
