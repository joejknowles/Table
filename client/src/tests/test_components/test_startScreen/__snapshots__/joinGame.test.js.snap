exports[`test component renders as before 1`] = `
<div
  className="join-game">
  <input
    onChange={[Function]} />
  <div>
    <Unknown />
    <Unknown />
  </div>
</div>
`;
